chapters/kodi/index.md

```diff
@@ -0,0 +1,269 @@
+# *Architecting Software to Keep the Lazy Ones On the Couch*
+
+*A Report On the Journey of Team Kodi*
+
+<p align="center">
+  <img src="images/Kodi.png" width=700px>
+</p>
+
+*Authors: [Kaj Dreef](github.com/kajdreef), [Menno van der Reek](github.com/mvanderreek), [Koen Schaper](github.com/kpschaper), [Maarten Steinfort](github.com/maartensteinfort)*
```

Comment:
And add a `https://`otherwise github will try to resolve this as `delftswa.github.io/chapters/kodi/github.com/kpschaper`